XDMA/linux-kernel/xdma/xdma_mod.c

```diff
@@ -45,6 +45,12 @@ MODULE_LICENSE("Dual BSD/GPL");
 static int xpdev_cnt;
 
 static const struct pci_device_id pci_ids[] = {
+	// Technovative Applications firmware for VPX552
+	{ PCI_DEVICE(0x5441, 0x2022), },	// 0x5441 = "TA" 
+
+	// Vadatech firmware for VPX552 
+	{ PCI_DEVICE(0xABCD, 0xC552), },
```

Comment:
Out of curiosity, did you register to get this vendor id? 

(I'm not part of xilinx, just an other user)